docs/framework/development-tools/package-generator/javascript-package.md

```diff
@@ -124,7 +130,22 @@ Examples:
 npm run stylelint
 ```
 
-### `dll:build`
+### `build:dist`
+
+Creates npm and browser builds of your plugin. These builds can be added to the editor by following the {@link updating/new-installation-methods Migrating CKEditor&nbsp;5 to new installation methods} guide.
```

Comment:
The regular guide should not link to the migration guide. It should be the other way around, because regular guides will be updated in the future, while migration guides will not.

Reply:
Any suggestion on where to link it, then? I would rather not link back to legacy, but there are few other places that could work here.